api/src/controllers/login.js

```diff
@@ -204,6 +201,18 @@ const renderTokenLogin = (req, res) => {
     .then(body => res.send(body));
 };
 
+const getUserCtxRetry = async (options, retry = 10) => {
+  try {
+    return await auth.getUserCtx(options);
+  } catch (err) {
+    if (retry > 0) {
```

Comment:
Should we check if the error was a `404` before retrying?  (Also is it a `404` error or `401` that we would get?  PR description says `404`...)

Also, can you add a brief in-line comment here to explain why we think we might get a different response when retrying?